### PR TITLE
Fixes #8762: The Category list is cut on small screen

### PR DESCRIPTION
**Pull request by RaphaelGauthier**
https://www.rudder-project.org/redmine/issues/8762

### CODE REVIEW

**Comment by VinceMacBuche** [comment]
This is a really very nice responsive chnage !! But it miss the initial goal of the issue !

It that you can't access to some  generic method categories in the add method screen when screen height is too small

**Comment by VinceMacBuche** [comment]
It can be scrolled on small screen but still not on normal screen ;)

File: tools/ncf.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
 additional_path = ["/opt/rudder/bin","/usr/sbin","/usr/local"]
 
 # Verbose output
-VERBOSE = 0
+VERBOSE = 1
```
Comment (VinceMacBuche) [inline tools/ncf.py]: Set back to non verbose mode please

File: tools/ncf.py [hunk 2]
```diff
@@ -79,7 +79,7 @@ def check_output(command, env = {}):
     sys.stderr.write(error)
   else:
     if VERBOSE == 1:
-      sys.stderr.write("VERBOSE: Exception triggered, Command returned error code " + retcode + "\n")
+      sys.stderr.write("VERBOSE: Exception triggered, Command returned error code " + str(retcode) + "\n")
```
Comment (VinceMacBuche) [inline tools/ncf.py]: Good catch !!

**Comment by RaphaelGauthier** [comment]
PR updated

**Comment by RaphaelGauthier** [comment]
PR updated

File: builder/index.html [hunk 3]
```diff
@@ -394,16 +391,15 @@ <h4 class="panel-title">
                             </li>
                         </ul>
                     </div>
-                    <div class="col-md-3" class="bs-sidebar navbar" >
-                        <ul class="nav bs-sidenav" style="position:fixed; margin-top:68px;">
+                    <div class="col-lg-2 col-md-2 col-sm-2 col-xs-4 ncf_col" class="bs-sidebar navbar" ng-if="selectedMethod === undefined">
```
Comment (VinceMacBuche) [inline builder/index.html]: I think you could have col-lg-1 col-md1 (and inscrease on to col-lg-4 on the methods) and reduce the padding on that column.

It's better to have more place on the left side

Comment (RaphaelGauthier) [inline builder/index.html]: This is what happens when I set col-lg-1 to the "Categories" list :

col-lg-1

It works properly on very large screen ( width > ~1450px), but below this width, some categories names are larger than the column width :/

Comment (VinceMacBuche) [inline builder/index.html]: You can gain like 20 / 25px 

It's not a problem if the lines are cut at the end

Comment (RaphaelGauthier) [inline builder/index.html]: Oh, I didn't see... I'm going to try that !

**Comment by RaphaelGauthier** [comment]
PR updated

File: builder/index.html (hunk 3 above)
Comment (RaphaelGauthier) [inline builder/index.html]: I remove the column padding, and reduce to 10px the categories names padding. That works pretty well, thanks !

**Comment by RaphaelGauthier** [comment]
PR updated

**Comment by RaphaelGauthier** [comment]
PR updated

**Pull request merged**